src/pages/index.js

```diff
@@ -44,12 +68,19 @@ function openProfilePopup() {
 
 function changeProfileInfo(event, data) {
   user.setUserInfo(data);
```

Comment:
You need to update `DOM` only in `then` blocks after a successful response from the server so that when there is an error, it shouldn’t change

**Please, correct it everywhere in the code**